doc/source/language/variables.rst

```diff
@@ -1,27 +1,77 @@
-.. _variables:
-
 Variables & Statements
 ======================
 
 .. contents::
     :local:
-    :depth: 1
+    :depth: 2
 
 .. _declare:
 
-``DECLARE``
------------
+``DECLARE .. TO``
+-----------------
+
+Syntax: ``DECLARE`` *identifier* ``TO`` *expression* *dot*
+
+.. warning::
+    .. versionadded:: 0.17
+        ***BREAKING CHANGE**
+        The meaning, and syntax, of this statement changed considerably
+        in this update.  Prior to this version, DECLARE always created
+        global variables no matter where it appeared in the script.
+        See 'initialier required' below.
+
+Declares a variable that is limited in scope to the code block it appears
+in, and gives it an initial value::
+
+    DECLARE X TO 1.
+
+(A "code block" is any section of statements that begins with a
+left-curly-brace "{" and ends with its closing right-curly-brace "}".)
+
+Any variable declared with DECLARE will only exist inside the code block
+section it was created in.  After that code block is finished, the variable
+will no longer exist.
+
+.. note::
+    There is an implicit outer scope block around a whole kerboscript
+    program, such that if you use a DECLARE .. TO statement inside
+    a script, it will be one nesting level "higher" than the implicit
+    globals you make with SET.  In other words, using DECLARE always
+    makes a variable that is at least limited to the program script
+    it is inside of.
```

Comment:
Arrrg.  This will have to be edited again also.  This isn't true anymore.  Now file scope and global scope are the same.